scripts/release/monitor/monitor_release/settings.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+from dotenv import load_dotenv
+from pydantic import BaseModel
+
+
+class Settings(BaseModel):
+    portainer_url: str
+    portainer_username: str
+    portainer_password: str
+    starts_with: str
+    swarm_stack_name: str
+    portainer_endpoint_version: int
+
+
+def get_settings(deployment):
+    load_dotenv(".env")
+
+    if deployment == "master":
+        portainer_url = os.getenv("MASTER_PORTAINER_URL")
+        portainer_username = os.getenv("MASTER_PORTAINER_USERNAME")
+        portainer_password = os.getenv("MASTER_PORTAINER_PASSWORD")
+
+        return Settings(
+            portainer_url=portainer_url,
```

Comment:
no need for this, you can load dotenvs from pydantic directly! 
https://docs.pydantic.dev/1.10/usage/settings/#dotenv-env-support